recipes/kcodecs/all/conanfile.py

```diff
@@ -0,0 +1,64 @@
+import os
+from conans import ConanFile, CMake, tools
+
+class KcodecsConan(ConanFile):
+    name = "kcodecs"
+    license = (
+        "BSD-3-Clause", "GPL-2.0-or-later", "LGPL-2.0-only", "LGPL-2.0-or-later", 
+        "LGPL-2.1-or-later", "MIT", "MPL-1.1")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://api.kde.org/frameworks/kcodecs/html/index.html"
+    topics = ("conan", "kde", "encodings", "strings")
+    description = "KCodecs provide a collection of methods to manipulate strings using various encodings."
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False]
+    }
+    default_options = {'shared': False, "fPIC": True}
+    exports_sources = ["CMakeLists.txt"]
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def build_requirements(self):
+        self.build_requires("extra-cmake-modules/5.75.0")
+
+    def requirements(self):
+        self.requires("gperf/3.1")
+        self.requires("qt/5.15.1")
```

Comment:
Not available yet: https://github.com/conan-io/conan-center-index/pull/1759

Reply:
Im preparation this until https://github.com/conan-io/conan-center-index/pull/3211 and https://github.com/conan-io/conan-center-index/pull/1759 get merged